README.md

```diff
@@ -35,104 +33,92 @@ Note: Just a heads up, the Swift Package Manager approach isn't ready for use ju
     - `pod 'Transcend'`
 - Run `pod install` to fetch the Transcend Framework.
 
-### Step 2: Use the custom Transcend WebView (Share the AirgapUrl)
+### Using Swift Package Manager
 
-- Application developers collecting consent with our WebView have the flexibility to employ this view in various contexts based on their application's logic. They can integrate this view during their application's startup view , or in response to a button click event. To use Transcend’s webView in SwiftUI the following changes are required:
-    
-    ### Documentation for TranscendConsentWebViewUI
-    
-    ```swift
-    // For iOS.V>=13.0 we expose TranscendConsentWebViewUI 
-    @available(iOS 13.0, *)
-    public struct TranscendWebViewUI : View {
-    
-        public var transcendConsentUrl: String
-    
-        public init(transcendConsentUrl: String)
-    
-        /// The content and behavior of the view.
-        ///
-        /// When you implement a custom view, you must implement a computed
-        /// `body` property to provide the content for your view. Return a view
-        /// that's composed of built-in views that SwiftUI provides, plus other
-        /// composite views that you've already defined:
-        ///
-        ///     struct MyView: View {
-        ///         var body: some View {
-        ///             Text("Hello, World!")
-        ///         }
-        ///     }
-        ///
-        /// For more information about composing views and a view hierarchy,
-        /// see <doc:Declaring-a-Custom-View>.
-        @MainActor public var body: some View { get }
-    
-        /// The type of view representing the body of this view.
-        ///
-        /// When you create a custom view, Swift infers this type from your
-        /// implementation of the required ``View/body-swift.property`` property.
-        public typealias Body = some View
-    }
-    ```
-    
-    ### Usage of TransendConsentWebViewUI
-    
-    ```swift
+```
+    https://github.com/transcend-io/Transcend-spm-sdk.git
+```
+
+## Usage
+Please consult the documentation [here](https://docs.transcend.io/docs/consent-management/mobile-consent/ios) for more details.
+
+### Initialization of API instance 
+
+The referrence for API instance in this repository can be found [here](https://github.com/transcend-io/consent-manager-ios-sample-sdk/blob/dev/sampleSDK/sampleSDKApp.swift#L45)
+- Note: This will not render Consent Banner on UI
```

Comment:
could you add more detail here? im not really sure what this means

Reply:
Ahh!! Added few more details. Its basically, the instance that allows you to talk to airgap.js api which is hidden behind all the views. User calls `getRemies` or `getConsent` before showing the UI instance